partcad/src/partcad/runtime_python.py

```
@@ -19,10 +19,10 @@
 from . import sync_threads
 
 
-class VenvLock(object):
+class VenvLock:
     def __init__(self, runtime, venv: str):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)